src/vanilla/utils/atomFamily.ts

```diff
@@ -47,6 +47,10 @@ export function atomFamily<Param, AtomType extends Atom<unknown>>(
     return newAtom
   }
 
+  createAtom.getParams = () => {
+    return Array.from(atoms.keys())
```

Comment:
Maybe we should just return the iterator:
```suggestion
    return atoms.keys()
```